src/cli.ts

```diff
@@ -479,4 +492,30 @@ async function withdraw_getHash(ctx: Context, to: string, amount: number, id: st
 async function withdraw_useSignature(ctx: Context, id: string) {
   const txId = await sendSignedWithdrawalTransaction(ctx, id)
   logSuccess(`Transaction with id ${txId} sent to the node`)
-}
\ No newline at end of file
+}
+
+
+//////////////////////////////////////////////////////////////////////////////////////////
+// Transaction signing
+

```

Comment:
can you add proper documentation as comment. As we done for others

Reply:
Removed export for this function since we don't need this anywhere else. Comment is just to visually differentiate between all functions in this file as is being followed everywhere else